mbtools/validate_mbz_html.py

```diff
@@ -26,7 +26,9 @@
                        "https://www.youtube.com",
                        "https://youtu.be",
                        "https://characterlab.org/",
-                       "https://digitalpromise.org"]
+                       "https://digitalpromise.org",
+                       "https://www.doe.virginia.gov/instruction"
```

Comment:
@mjoss17 - We can probably just make this ""https://www.doe.virginia.gov" just in case they reference other resources later from the same domain.